chapters/en/slides/module2/module2_01.md

```diff
@@ -0,0 +1,184 @@
+Notes: So far we have seen how to use points and lines to represent data
+visually. In this lecture, we will see how we can also use areas and
+bars for this purpose. Before we dive in to the code, let’s discuss how
+different visual channels, such as position, area, etc, can impact how
+easy it is for us to accurately interpret the plotted data.
+
+As in many cases, an efficient way to learn this topic is through
+personal experience. Please follow along in the exercises on the next
+two slides to see which visual channels you find the most effective.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/circle-comparison.png)
+
+Notes: In the image in this slide, try to estimate how many times larger
+the big circle is compared to the small one.
+
+---
+
+## Comparing sizes is easier for some geometrical objects than for others
+
+![](/module2/bar-comparison.png)
+
+Notes: In the image in this slide, try to estimate how many times larger
+the big bar is compared to the small one.
+
+---
+
+## Both the circles and rectangles differ by 7x
+
+<img src="/module2/7x.png" style="width:60.0%" />
+
+Notes: In both cases the answer is seven times bigger. Even if you
+guessed both of these exactly correct, most people find it is much
+easier to compare the length or position of the bars rather than the
+area of the circles. For the circles, you might even have hesitated at
+exactly what to compare when we said “how many time *larger*”, were you
+supposed to compare the area or the diameter? This is less ambiguous for
+bars as long as their widths are kept the same.
+
+This is important to keep in mind, especially when communicating to
+others via visualization, but also when creating plots for yourself.
+These two examples are originally from [Jeffrey Heer’s PyData
+talk](https://www.youtube.com/watch?v=hsfWtPH2kDg), who is visualization
+researcher at the university of Washington and whose research group
+created D3 and VegaLite (which Altair is based on).
+
+---
+
+## Summary of visual channel efficiency
+
+<img src="/module2/visual-channels-rank.png" style="width:80.0%" />
+
+Notes: Even if you got both these right yourself, the fact that many
+people prefer one over the other means that in order for you to create
+effective visualizations you need to know which visual channels are the
+easiest for humans to decode. Luckily, there has been plenty of research
+in this area, which can be summarizes in the schematic on this slide.
+
+Position is by far the best and therefore we should put our most
+important comparison there. Using position often means that we can’t use
+other things such as length or angle (like the angle in a pie chart),
+but we can add size or color to represent other relationships. Even if
+it is hard to tell exact information from these (is this color/dot 2x
+darker/bigger than another?) they are good to give and idea of trends in
+the data (are these dots darker/bigger than the others?
```

Comment:
You can combine position and angle (think wind direction visualization). I think a pie chart is more of an area thing. I think this second paragraph needs some work to make it clear and effective.

Reply:
Good point, I guess wind/vector field has ange, position, and also length. Such a rich viz, I should try use it more often! 

Regarding pie charts, they seem to be classified as "angle" in the viz studies I have see references to ([cited here](https://socviz.co/lookatdata.html#visual-tasks-and-decoding-graphs)), e.g. in a study from the D3 author with this schematic:

![image](https://user-images.githubusercontent.com/4560057/105269934-bc9bcf00-5b49-11eb-9079-ca0f23de45a9.png)
 
But I agree with you that there is an area interpretation as wel, so not sure which is dominant. For this slide it would make sense to ref the tiling line as in the picture instead for angle. I will clarify and also make more effective overall.